### ISSUE TITLE
NPM install error

### ISSUE THREAD

**Issue by markplewis** [dorelljames/sanity-plugin-asset-source-pexels#18]
Running `npm install --save sanity-plugin-asset-source-pexels` gives me the following error:

```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:.yalc/sanity-plugin-asset-source-pexels
```

I'm running Sanity `3.8.2` with Node `18.13.0` and NPM `8.19.3` on macOS `13.3`.

**Comment by markplewis** [comment]
Also, when I run `sanity install asset-source-pexels`, I get the following error:

```
Running 'npm install --legacy-peer-deps --save asset-source-pexels'
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/asset-source-pexels - Not found
npm ERR! 404
npm ERR! 404  'asset-source-pexels@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by dorelljames** [comment]
Fixed in version [3.0.1](https://www.npmjs.com/package/sanity-plugin-asset-source-pexels/v/3.0.1). 

Can you please try again with:

```bash
npm install --save sanity-plugin-asset-source-pexels
```

Thanks!

**Comment by markplewis** [comment]
Great! Thank you, @dorelljames. It's working now.
